archivers/par2/Portfile

```diff
@@ -17,20 +18,18 @@ license           GPL-2
 maintainers       jabber.org:julian
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
```

Comment:
the homepage is set automatically by the github PG so you can rove this line

Reply:
please also fix the lint warning: `Line 20 has trailing whitespace before newline`

Reply:
Done.